content/knowledge-base/guides/connection-details.md

```diff
@@ -0,0 +1,593 @@
+---
+title: Understanding Connection Details
+weight: 11
+description: "How to create and manage connection details across Crossplane managed resources, composite resources, Compositions and Claims"
+---
+
+Using connection details in Crossplane requires the following components:
+* Defining the `writeConnectionSecretToRef.name` in a [Claim]({{<ref "/master/concepts/claims#claim-connection-secrets">}}).
+* Defining the `writeConnectionSecretsToNamespace` value in the [Composition]({{<ref "/master/concepts/compositions#composite-resource-combined-secret">}}).
+* Define the `writeConnectionSecretToRef` name and namespace for each resource in the
+  [Composition]({{<ref "/master/concepts/compositions#composed-resource-secrets">}}).
+* Define the list of secret keys produced by each composed resource with `connectionDetails` in the
+  [Composition]({{<ref "/master/concepts/compositions#define-secret-keys">}}).
+* Optionally, define the `connectionSecretKeys` in a 
+  [CompositeResourceDefinintion]({{<ref "/master/concepts/composite-resource-definitions#manage-connection-secrets">}}).
+
+{{<hint "note">}}
+This guide discusses creating Kubernetes secrets.  
+Crossplane also supports using external secret stores like [HashiCorp Vault](https://www.vaultproject.io/). 
+
+Read the [external secrets store guide]({{<ref "/knowledge-base/integrations/vault-as-secret-store">}}) for more information on using Crossplane
+with an external secret store. 
+{{</hint >}}
+
+## Background
+When a [Provider]({{<ref "/master/concepts/providers">}}) creates a managed
+resource, the resource may generate resource-specific details. These details can include 
+usernames, passwords or connection details like an IP address.  
+
+Crossplane refers to this information as the _connection details_ or 
+_connection secrets_.   
```

Comment:
These two paragraphs should better be right at the top to set the stage what this topic is actually about.

Reply:
The reason it's not is because the current top is intended to be a quick reference for anyone dealing with secrets and wanting a checklist to see what they did right/wrong.